app/lib/utils.dart

```diff
@@ -51,3 +51,56 @@ String canonicalizeVersion(String version) {
   }
   return canonicalVersion.toString();
 }
+
+RegExp _identifierExpr = new RegExp(r'^[a-zA-Z0-9_]*$');
```

Comment:
Replace `*` by `+`, we already know it's not the empty string (which is handled before).